locksmith/__tests__/controllers/v2/verifierController.test.ts

```diff
@@ -0,0 +1,67 @@
+import { ethers } from 'ethers'
+import request from 'supertest'
+
+const app = require('../../../src/app')
+
+jest.setTimeout(600000)
+const lockAddress = '0x3F09aD349a693bB62a162ff2ff3e097bD1cE9a8C'
+
+describe('Verifier v2 endpoints for locksmith', () => {
```

Comment:
You need to login and use the authorization header. 

Also only user Address which is a lock manager can be used here afaik.